README.md

```diff
@@ -1,9 +1,11 @@
 # issue-sync
 
-Issue-sync is a tool for synchronizing GitHub and JIRA issues. It grew
+Issue-sync is a tool for pushing GitHub issues to JIRA. It grew
```

Comment:
Nit: maybe instead of pushing we can clarify “synchronizing issues from GitHub to JIRA” or “mirroring issues from ... in ...”

Reply:
I like the mirroring suggestion.

```suggestion
Issue-sync is a tool for mirroring GitHub issues in JIRA. It grew
```